src/sider/arbeidsuforhet/OppfyltForm.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import { BodyShort, Box, Button, Heading, Textarea } from "@navikt/ds-react";
+import { ButtonRow } from "@/components/Layout";
+import { SkjemaInnsendingFeil } from "@/components/SkjemaInnsendingFeil";
+import { Forhandsvisning } from "@/components/Forhandsvisning";
+import { useForm } from "react-hook-form";
+import {
+  VurderingRequestDTO,
+  VurderingType,
+} from "@/data/arbeidsuforhet/arbeidsuforhetTypes";
+import { useArbeidsuforhetVurderingDocument } from "@/hooks/arbeidsuforhet/useArbeidsuforhetVurderingDocument";
+import { useSendVurderingArbeidsuforhet } from "@/data/arbeidsuforhet/useSendVurderingArbeidsuforhet";
+
+const texts = {
+  title: "Skriv en kort begrunnelse for hvorfor bruker oppfyller § 8-4",
+  info: "Det du skriver her blir liggende i historikken på brukeren og bruker vil få innsyn i notatet om de har behov.",
```

Comment:
Også usikker på om vi vil bruke ordet `notatet` her 🤔

Reply:
🤷‍♂️

Reply:
Jeg lagde en lapp på tekstene her, så kan vi fikse sånne småting for seg 👍